airflow-core/src/airflow/config_templates/config.yml

```
@@ -1405,6 +1405,13 @@ api:
       version_added: 2.7.0
       example: ~
       default: "False"
+    grid_view_sorting_order:
+      description: |
+        Sorting order in grid view. Valid values are: ``topological``, ``hierarchical_alphabetical``
+      version_added: 2.7.0
```

```suggestion
      version_added: 3.1.0
```

Probably? Shouldn't be 2.7.0 though.

---

Yes, I thought about that, the config was not added in 3.1.0 but moved in 3.1.0 and the original param that was added in 2.7.0 does not really exist anymore....

I decided to go with 2.7.0 because this is how other param are handled. In the api section there are a lot of `version_added` 2.7.0, 2.2.0, 2.x.0, but really nothing in the `api` was created before `3.0.0` so I figured this is how we are doing it

---

I think we havent been consistent then - all the stuff moved into the new dag processor section had these removed :)

It's fine for this one, but we might want to do a pass so we are consistent across the board. Curious what others think is the right answer.

---

I wasn't aware of that, yes it might be good to be consistent.

---

We can easily do a pass for the `api` and `dagprocessor` section. (those are new and easy to update as a bulk)

---

Added to my ever growing todo list, (but somewhere near the top 😄)